lib/file-helper.js

```diff
@@ -73,3 +73,23 @@ exports.getDirectories = itemPath => {
   }
   return directories
 }
+
+// This helper function takes a destination path and
+// returns creates all directories for that path in they're not present
+exports.createDirectoriesForPath = (targetDir, {isRelativeToScript = false} = {}) => {
```

Comment:
Do we need to make this sync? Looks like the place we're using this function would allow for an async implementation of this.

Reply:
wouldn't the file likely be tried to be create before the parent directories are and throeing an error in that case?

Reply:
the way the function was written it must be sync; if we convert it to async (`mkdir`) we should use callbacks